odoo/addons/base/models/res_partner_bank.py

```diff
@@ -0,0 +1,219 @@
+import re
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+def sanitize_account_number(account_number):
+    if account_number:
+        return re.sub(r'\W+', '', account_number).upper()
+    return False
+
+
+class ClearingLabel(models.Model):
+    _name = 'clearing.label'
```

Comment:
@kmagusiak should we prefix this with `res`?
if so, should we move this model to a new file?